crates/ra_hir/src/code_model.rs

```diff
@@ -394,6 +395,10 @@ impl Struct {
         Type::from_def(db, self.id.lookup(db.upcast()).container.module(db.upcast()).krate, self.id)
     }
 
+    pub fn is_packed(self, db: &dyn HirDatabase) -> bool {
```

Comment:
Would it make more sense to expose a `ReprData` and forward the info along, rather than filter to just packed?

Reply:
I think I'd expose `ReprKind` for now

Reply:
Done